src/IRMatch.h

```diff
@@ -930,17 +930,17 @@ std::ostream &operator<<(std::ostream &s, const BinOp<Mod, A, B> &op) {
 }
```

Comment:
IIUC std::forward is supposed to be strictly better. However, this whole file is very sensitive to what the compiler decides needs an address and stack location and what doesn't, so this change is not a no-brainer. I need to check what the stack frame looks like for some of the simplify visitors before and after this change.

Reply:
I don't consider this change really essential... basically just wanted to grind through the new checks while waiting on some buildbots. This can wait for a lazy day. (Or be merged piecewise.)